dataservice/api/genomic_file/README.md

```diff
@@ -0,0 +1,8 @@
+prefix: `GF`
+
+Genomic Files track files related to the bioinformatics process such as `bam`
+`cram`, `gvcf`, and other files. Files may be related to a `Biospecimen`
+(eg: an aligned `cram`), others may not be directly related to `Biospecimen`,
+having been derived from some other file(s) (eg: a cohort `vcf`).
+Most all `Genomic Files` should be able to resolve to one or more `Biospecimens`
```

Comment:
I think this last sentence is a little confusing. Don't all of our unharmonized and harmonized genomic files have links to a Biospecimen through the BiospecimenGenomicFile relation? Maybe its more like, most genomic files should be able to resolve to one or more specimens through the BiospecimenGenomicFile, and others can be resolved using the Task table. And then maybe add something about Task - Task is an executed bioinformatic process which produced Genomic Files from one or more input Genomic Files which are linked to existing Biospecimens.